src/components/Login/Login.jsx

```diff
@@ -76,7 +76,7 @@ export class Login extends React.Component {
                     </div>
                     <Monster />
                     <form className={styles.loginForm} onSubmit={this.sendLogin.bind(this)}>
-                        <h3 data-cy="login-error" className={styles.errorText}>{}</h3>
+                        <h3 data-cy="login-error" className={styles.errorText}>{(typeof this.props.error === 'string') ? this.props.error : ""}</h3>
```

Comment:
out of curiosity what other types might the error be?

Reply:
I think it's an object when it's produced from the axios request

Reply:
Can't it just start as an empty string and then set the string ourselves when we receive an error from an axios request?